README.md

```diff
@@ -1,30 +1,65 @@
-# my-component
+# Storage Extractor
 
-[![Build Status](https://travis-ci.org/keboola/my-component.svg?branch=master)](https://travis-ci.org/keboola/my-component)
+[![Build Status](https://travis-ci.org/keboola/ex-storage.svg?branch=master)](https://travis-ci.org/keboola/ex-storage)
 
-> Fill in description
+Extracts tables from source bucket from a source project. You need to provide a Storage
+token (which identifies both the project and the bucket) from the source project which 
+has `read` access to the source bucket **only**.
 
 # Usage
 
-> fill in usage instructions
+Configuration:
+
+```
+{
+	"parameters": {
+		"#token": "some-token",
+		"url": "https://connection.keboola.com/",
+		"tableName": "some-table"
+	}
+}
+```
+
+For incremental loading, you the `changedSince` parameter can be provided:
+
+```
+{
+	"parameters": {
+		"#token": "some-token",
+		"url": "https://connection.keboola.com/",
+		"tableName": "some-table",
+		"changedSince": "-1 day"
+	}
+}
+```
+
+`changedSince` can be a timestamp or anything which 
+[`strtotime`](http://php.net/manual/en/function.strtotime.php) can understand.
 
 ## Development
- 
+
 Clone this repository and init the workspace with following command:
 
 ```
-git clone https://github.com/keboola/my-component
-cd my-component
+git clone https://github.com/keboola/ex-storage
+cd ex-storage
 docker-compose build
 docker-compose run --rm dev composer install --no-scripts
 ```
 
 Run the test suite using this command:
 
 ```
-docker-compose run --rm dev composer tests
+docker-compose run --rm dev composer ci
 ```
- 
+
+The following environment variables have to be set:
+
+- KBC_TEST_URL - URL of the source Storage (e.g. https://connection.keboola.com/)
+- KBC_TEST_BUCKET - Bucket in the source project
+- KBC_TEST_TOKEN - Token to the source project (with read access to the source bucket)
+- KBC_TEST_WRITE_TOKEN - Token to the source project (with write access to the source bucket)
```

Comment:
tyjo, tady by se zase hodil manageapi token do nějaký testovací kbc instance, kde můžeme zakládat a zahazovat projekty pro každej test.

Reply:
Imho stačí testovací organizace a do ní dát testovacího usera jehož manage token pak použiješ. Separátní KBC stack pro tyhle účely mi přijde zbytečný a proti testingu by to určitě jet nemělo protože tam zrovna může být rozbitá/nepublikovaná verze.

Reply:
šlo mi asi jen o to, ať se zbytečně nezaplácávaj zdroje, nelítaj idčka projektů zbytečně nahoru, ale ono to asi ničemu zásadně nevadí, možná naopak :-)

Reply:
no tak kdyby tam byl mng token, tak naopak budes vytvaret 2 tokeny pro kazdy spusteni testu, takhle jsou tam furt 2. A vytvaret celou instanci kvuli celkem primitivnimu extractoru mi prijde jako overoverkill :)

Reply:
jasně, ale vytvořit jeden token manage token zabere jen chvilku, nasetupovat správně dva tokeny (a před nima ještě bucket), to už není úplně brnkačka